From 92d4b6c80ed91c2f569392ed0f0076fa5056c6f0 Mon Sep 17 00:00:00 2001
From: Punnaiah Choudary Kalluri <punnaia@xilinx.com>
Date: Mon, 7 Apr 2014 19:46:51 +0530
Subject: [PATCH 342/793] spi: zynq-qspi: Removed dead code

Remove check for byte_to_receive < 0 as this is unecessary.

Signed-off-by: Punnaiah Choudary Kalluri <punnaia@xilinx.com>
Signed-off-by: Harini Katakam <harinik@xilinx.com>
Signed-off-by: Michal Simek <michal.simek@xilinx.com>
---
 drivers/spi/spi-zynq-qspi.c |    6 ------
 1 files changed, 0 insertions(+), 6 deletions(-)

diff --git a/drivers/spi/spi-zynq-qspi.c b/drivers/spi/spi-zynq-qspi.c
index 72e5eb4..bfaaca6 100644
--- a/drivers/spi/spi-zynq-qspi.c
+++ b/drivers/spi/spi-zynq-qspi.c
@@ -231,8 +231,6 @@ static void zynq_qspi_copy_read_data(struct zynq_qspi *xqspi, u32 data, u8 size)
 		xqspi->rxbuf += size;
 	}
 	xqspi->bytes_to_receive -= size;
-	if (xqspi->bytes_to_receive < 0)
-		xqspi->bytes_to_receive = 0;
 }
 
 /**
@@ -253,8 +251,6 @@ static void zynq_qspi_copy_write_data(struct zynq_qspi *xqspi, u32 *data,
 	}
 
 	xqspi->bytes_to_transfer -= size;
-	if (xqspi->bytes_to_transfer < 0)
-		xqspi->bytes_to_transfer = 0;
 }
 
 /**
@@ -470,8 +466,6 @@ static irqreturn_t zynq_qspi_irq(int irq, void *dev_id)
 							ZYNQ_QSPI_RXD_OFFSET);
 				}
 				xqspi->bytes_to_receive -= 4;
-				if (xqspi->bytes_to_receive < 0)
-					xqspi->bytes_to_receive = 0;
 			}
 			rxindex++;
 		}
-- 
1.7.5.4

